src/components/footer/footer.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+import Icon from './icon';
+import facebook from './facebook.svg';
+import github from './github.svg';
+import medium from './medium.svg';
+import twitter from './twitter.svg';
+
+const styles = theme => ({
+	container: {
+		backgroundColor: '#352A3A'
+	},
+	buttonLink: {
+		textDecoration: 'none'
+	},
+	content: {
+		maxWidth: theme.maxWidth,
+		padding: theme.spacing(1, 3, 1, 3),
+		margin: 'auto',
+		display: 'flex',
+		flexDirection: 'row',
+		justifyContent: 'space-between',
+		alignItems: 'center',
+		fontFamily: ['Poppins', 'sans-serif'],
+		fontSize: theme.typography.fontSize,
+		color: '#FFFFFF',
+		textAlign: 'center',
+		[theme.breakpoints.down('xs')]: {
+			flexDirection: 'column'
+		}
+	},
+	icons: {
+		display: 'flex',
+		justifyContent: 'space-between',
+		alignItems: 'center',
+		paddingTop: theme.spacing()
```

Comment:
I assume that this by default means `theme.spacing(1)`. But I prefer explicitness here.